src/test/javascript/spec/component/assessment-shared/assessment-locks.component.spec.ts

```diff
@@ -31,24 +50,64 @@ describe('AssessmentLocksComponent', () => {
                 { provide: LocalStorageService, useClass: MockSyncStorage },
                 { provide: SessionStorageService, useClass: MockSyncStorage },
                 { provide: TranslateService, useClass: MockTranslateService },
+                { provide: textAssessmentService, useClass: TextAssessmentService },
+                { provide: programmingAssessmentService, useClass: ProgrammingAssessmentManualResultService },
+                { provide: fileUploadAssessmentService, useClass: FileUploadAssessmentService },
+                { provide: modelingAssessmentService, useClass: ModelingAssessmentService },
             ],
-        }).compileComponents();
+        })
+            .compileComponents()
+            .then(() => {
+                programmingAssessmentService = TestBed.inject(ProgrammingAssessmentManualResultService);
+                textAssessmentService = TestBed.inject(TextAssessmentService);
+                fileUploadAssessmentService = TestBed.inject(FileUploadAssessmentService);
+                modelingAssessmentService = TestBed.inject(ModelingAssessmentService);
+            });
     });
 
     beforeEach(() => {
         fixture = TestBed.createComponent(AssessmentLocksComponent);
         component = fixture.componentInstance;
         fixture.detectChanges();
     });
-
     it('should create', () => {
         expect(component).toBeTruthy();
     });
 
     it('should call getAllLockedSubmissions on init', () => {
         component.ngOnInit();
-
         expect(component.getAllLockedSubmissions).toHaveBeenCalled;
```

Comment:
This isn't changed by you, but it looks like we're missing ``();`` here?

Reply:
I changed the code acrodingly. As far as i understand, this test did not do any testing anyways before